test/integrations/destinations/mp/processor/data.ts

```diff
@@ -2,6 +2,34 @@ const defaultMockFns = () => {
   jest.spyOn(Date, 'now').mockImplementation(() => new Date(Date.UTC(2020, 0, 25)).valueOf());
 };
 
+const sampleDestination = {
+  Config: {
+    apiKey: 'dummyApiKey',
+    token: 'dummyApiKey2',
+    prefixProperties: true,
+    useNativeSDK: false,
+  },
+  DestinationDefinition: {
+    DisplayName: 'Kiss Metrics',
+    ID: '1WhbSZ6uA3H5ChVifHpfL2H6sie',
+    Name: 'MIXPANEL',
+  },
+  Enabled: true,
+  ID: '1WhcOCGgj9asZu850HvugU2C3Aq',
+  Name: 'Kiss Metrics',
+  Transformations: [],
+};
+
+const getDestination = (configValues) => {
```

Comment:
Object.assign(sampleDestination, {Config: configValues}) -> does the same thing you are trying to do

Reply:
Instead of the above, we can try this to retain other properties as well which are not passed 
```
Object.assign({}, sampleDestination, {
    Config: Object.assign({}, sampleDestination.Config, configValues)
  });
```

Reply:
both are not behaving similarly?

Reply:
No, in the one you suggested only those properties will be there which are passed in configValues.

Reply:
Object.assign({}, sampleDestination, {Config: configValues}) , this should work.
example: Object.assign({}, {a: {b:2}, c:3}, {a: {b:3}}) -> { a: { b: 3 }, c: 3 }

Reply:
Yes working, should I go ahead and make changes for all tests?

Reply:
yes go ahead